runtime/CMakeLists.txt

```diff
@@ -122,6 +122,14 @@ foreach(PROG ${RUNTIME_PROGRAMS})
                  DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime/${BASEDIR})
 endforeach()
 
+globrecurse_wrapper(RUNTIME_QUERIES ${CMAKE_CURRENT_SOURCE_DIR} *.scm)
+
+foreach(QUERY ${RUNTIME_QUERIES})
+  get_filename_component(BASEDIR ${QUERY} DIRECTORY)
+  install_helper(FILES ${QUERY}
+                 DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/nvim/runtime/${BASEDIR})
+endforeach()
```

Comment:
are these distinct from the `RUNTIME_FILES` below, or couldn't `*.scm` just be added to its list of extensions?

Reply:
I thought it would be useful to have these as a separate step (at least for now). But I'm sure we could combine them if you think that's better?

Reply:
Well for now it is just duplicate code. it is better to separate then only if there is some extra logic (and there are note ye).